lighthouse-core/gather/computed/page-dependency-graph.js

```diff
@@ -173,26 +182,29 @@ class PageDependencyGraphArtifact extends ComputedArtifact {
           minCandidate = candidate;
           minDistance = distance;
         }
-      });
+      }
 
       if (!minCandidate) return;
       cpuNode.addDependency(minCandidate);
     }
 
+    /** @type {Map<string, CPUNode>} */
     const timers = new Map();
     for (const node of cpuNodes) {
       for (const evt of node.childEvents) {
         if (!evt.args.data) continue;
 
-        const url = evt.args.data.url;
```

Comment:
dataUrl makes me think of a data URI, which other url is this prefix disambiguating?

Reply:
purely style. I was wondering where `url` came from below, then got briefly confused by the various `forEach`es. Changed to `argsUrl`...wdyt? `evtUrl`? Can also revert.